File: server/public/model/limits.go
package model
type ServerLimits struct {
	MaxUsersLimit           int64 `json:"maxUsersLimit"`
	MaxUsersHardLimit       int64 `json:"maxUsersHardLimit"`
	ActiveUserCount         int64 `json:"activeUserCount"`
	SingleChannelGuestCount int64 `json:"singleChannelGuestCount"`
	SingleChannelGuestLimit int64 `json:"singleChannelGuestLimit"`
	PostHistoryLimit        int64 `json:"postHistoryLimit"`
	LastAccessiblePostTime  int64 `json:"lastAccessiblePostTime"`
}

File: webapp/platform/types/src/limits.ts
export type LimitsState = {
    serverLimits: ServerLimits;
};
export type ServerLimits = {
    activeUserCount: number;
    maxUsersLimit: number;
    maxUsersHardLimit?: number;
    lastAccessiblePostTime?: number;
    postHistoryLimit?: number;
    singleChannelGuestCount?: number;
    singleChannelGuestLimit?: number;
}